wom/src/main/scala/wom/types/WomFloatLike.scala

```diff
@@ -0,0 +1,19 @@
+package wom.types
+
+/** WomFloat actually wraps a double. Mimicking the structure of WomIntegerLike JIC we need to make the
+  * distinction between float and double that CWL seems to be suggesting we should make. */
+object WomFloatLike {
+  private def parseString(string: String): Double = {
```

Comment:
Not `ErrorOr[Double]`?

Reply:
If I understand what you're saying correctly, `ErrorOr`ing `coerce` would be a pretty significant change from the existing structure